android-exoplayer/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java

```diff
@@ -201,13 +201,8 @@ public void onCues(List<Cue> cues) {
 
         @Override
         public void onVideoSizeChanged(int width, int height, int unappliedRotationDegrees, float pixelWidthHeightRatio) {
-            boolean isInitialRatio = layout.getAspectRatio() == 0;
             layout.setAspectRatio(height == 0 ? 1 : (width * pixelWidthHeightRatio) / height);
-
-            // React native workaround for measuring and layout on initial load.
-            if (isInitialRatio) {
```

Comment:
With this patch you will have strange behavior.
If you are playing content with ABR, onVideoSizeChanged will be called on every time playlist change.
you need to check the aspect ratio have a real change,
something like this:
abs(oldAspectRatio - newAspectRatio) > aspectRatioChangeThreesold